groups/stack/profiles/development-eu-west-2/staging/vars

```diff
@@ -0,0 +1,38 @@
+aws_bucket = "staging-eu-west-2.terraform-state.ch.gov.uk"
+remote_state_bucket = "ch-service-staging-terraform-state"
+environment = "staging"
+deploy_to = "staging"
+state_prefix = "env:/staging"
+aws_profile = "staging-eu-west-2"
+
+cookie_domain = "companieshouse.gov.uk"
+cookie_name = "__SID"
+
+# Certificate for https access through ALB
+ssl_certificate_id = "arn:aws:acm:eu-west-2:250991044064:certificate/56b76dab-5728-4f83-a16d-e3dd59cd82c8"
+external_top_level_domain = "-staging-aws.companieshouse.gov.uk"
+internal_top_level_domain = "-internal-api-staging-aws.companieshouse.gov.uk"
+
+ec2_key_pair_name = "chs-aws-staging"
+
+# shared configs
+log_level = "TRACE"
+
+# eric configs
+eric_cache_url = "staging-chs-elasticache-001.3psdof.0001.euw2.cache.amazonaws.com:6379"
+
+# api configs
+internal_api_url = "https://internal-api-staging-aws.companieshouse.gov.uk"
```

Comment:
pretty sure this and all of the configs below that end with `companieshouse.gov.uk` should end with `company-information.service.gov.uk` instead